astropy/io/fits/convenience.py

```diff
@@ -646,12 +644,11 @@ def append(filename, data, header=None, checksum=False, verify=True, **kwargs):
 #
```

Comment:
It's totally unrelated to this PR, but why do we have this commented out section in the codebase? cc fits maintainers @saimn @MSeifert04 @drdavella?

Reply:
Sorry @bsipocz, it was me. I was experimenting and left a previous version of a code block commented out. It is gone now. My mistake.

Reply:
Oh, should have checked the full diff. Sorry :)